services/scanners/dns/dns_scanner.py

```diff
@@ -61,8 +62,14 @@ async def scan_dmarc(domain):
         # Extract the domain from the address string (e.g. 'dmarc@cyber.gc.ca' -> 'cyber.gc.ca').
         rua_domain = rua_addr.split("@", 1)[1]
 
-        # If the report destination differs from the originating domain, assert reports are being accepted.
-        if rua_domain == domain:
+        # Extract organizational domain from original domain (e.g. 'tracker.cyber.gc.ca' -> 'cyber.gc.ca')
+        extract = tldextract.TLDExtract(include_psl_private_domains=True)
+        extract.update()
+        parsed_domain = extract(domain)
+        org_domain = parsed_domain.domain + parsed_domain.suffix
```

Comment:
`'.'join([parsed_domain.domain, parsed_domain.suffix])`

I think we need to extract the organizational domains of both and compare them.